app/containers/TopicHighlighter/index.js

```diff
@@ -48,16 +48,23 @@ export class TopicHighlighter extends Component {
     let article = this.props.article;
     let topics = this.props.topics[current_article];
 
+    let loadingClass = '';
+
     if (this.props.nextArticle == undefined) {
       return (<div>DONE</div>) // TODO: Clean this up.
     }
 
+    if (this.props.article.isFetching) {
```

Comment:
you can use a ternary statement like here
https://github.com/Goodly/TextThresher/blob/master/app/components/Article/index.js#L40

😄